### ISSUE TITLE
Requests version

### ISSUE THREAD

**Issue by cardoza1991** [Torantulino/Auto-GPT#961]
### 

- [ ] I have searched the existing issues

### Steps to reproduce 🕹

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
docker 6.0.1 requires requests>=2.26.0, but you have requests 2.25.1 which is incompatible.
duckduckgo-search 2.8.5 requires requests>=2.28.2, but you have requests 2.25.1 which is incompatible.
gtts 2.3.1 requires requests<3,>=2.27, but you have requests 2.25.1 which is incompatible.
tiktoken 0.3.3 requires requests>=2.26.0, but you have requests 2.25.1 which is incompatible.

### Current behavior 😯

I came across this issue when I was installing requirements.txt is there a solution or is request==2.25.1 not good enough?

### Your prompt 📝

```yaml
# Paste your prompt here
```

**Comment by metacubic** [comment]
Try running with administrator privileges.

**Comment by ismaelgirard** [comment]
What do you mean by pip install requests==2.28.2 ?

**Comment by cardoza1991** [comment]
pip install requests has different versions the version I had before was version 2.28.1 and I changed it to version 2.5.1 to see if it fixed it but after changing to 2.28.2 it fixed the requests version error.

**Comment by p-i-** [comment]
Are you running in a fresh virtual environment?

```
> pip show requests
Name: requests
Version: 2.28.2
Summary: Python HTTP for Humans.
Home-page: https://requests.readthedocs.io
Author: Kenneth Reitz
Author-email: me@kennethreitz.org
License: Apache 2.0
Location: /Users/pi/code/2023/ML/devAGPT/.venv/lib/python3.11/site-packages
Requires: certifi, charset-normalizer, idna, urllib3
Required-by: docker, duckduckgo-search, google-api-core, gTTS, openai, pinecone-client, tiktoken
```

So you get 2.25.2 whereas I'm getting 2.28.2.

Likely you're using an old pip/python.

`pyenv` is a good tool for ensuring latest Python on your system.
